mmdet/datasets/crowdhuman.py

```diff
@@ -0,0 +1,131 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import json
+import logging
+import os.path as osp
+from typing import List, Union
+
+import mmcv
+from mmengine.fileio import dump, load
+from mmengine.logging import print_log
+from mmengine.utils import ProgressBar
+
+from mmdet.registry import DATASETS
+from .base_det_dataset import BaseDetDataset
+
+
+@DATASETS.register_module()
+class CrowdHumanDataset(BaseDetDataset):
+    r"""Dataset for CrowdHuman.
+
+    Args:
+        data_root (str): The root directory for
+        ``data_prefix`` and ``ann_file``.
+        ann_file (str): Annotation file path.
+    """
+
+    METAINFO = {
+        'CLASSES': ('person', ),
+        # PALETTE is a list of color tuples, which is used for visualization.
+        'PALETTE': [(220, 20, 60)]
+    }
+
+    def __init__(self, data_root, ann_file, **kwargs):
+        # id_hw file is an additional annotation information which record the
+        # size of each image. This file is automatically created when you
+        # first load the CrowdHuman dataset by mmdet
+        ann_file_name = osp.basename(ann_file)
+        if 'train' in ann_file_name:
+            self.id_hw_path = osp.join(data_root, 'id_hw_train.json')
```

Comment:
We can add download link of `id_hw_train.json` and `id_hw_val.json` in README

Reply:
I added a parameter to CrowdHumanDataset that allows users to manually set the path of id_hw file. 
This file can be achieved by three ways:
1. created by CrowdHumanDataset automatically.
2. created by  tools/misc/get_crowdhuman_id_hw.py manually.
3. download.